src/components/Layout/index.jsx

```diff
@@ -8,22 +8,22 @@ import "./styles.module.scss";
  * Block Layout
  */
 const Layout = ({ sidebar, PageComponent, ...routeProps }) => {
-  const [size, mainRef] = useTargetSize();
-  const asideRef = useRef();
-  const height = size && size.height;
+  // const [size, mainRef] = useTargetSize();
```

Comment:
we generally don't leave commented out code. we either delete it entirely or add a comment about when the code should be uncommented. for example, if there is a future ticket that will use the code, then we can say 
`// this will be used when implementing PROD-###`

but in this case, we aren't planning on ever changing this back, so let's just delete it.